wp-parsely.php

```diff
@@ -1046,15 +1071,65 @@ public function insert_parsely_javascript() {
 		/**
 		 * Filters whether to include the Parsely JavaScript file.
 		 *
-		 * If true, the file is included.
+		 * If true, the JavaScript files are sourced.
 		 *
 		 * @since 2.2.0
 		 *
 		 * @param bool $display True if the JavaScript file should be included. False if not.
 		 */
-		if ( apply_filters( 'parsely_filter_insert_javascript', $display ) ) {
-			include 'parsely-javascript.php';
+		if ( ! apply_filters( 'parsely_filter_insert_javascript', $display ) ) {
+			return;
+		}
+
+		wp_register_script(
+			'wp-parsely-api',
+			plugin_dir_url( __FILE__ ) . 'build/init-api.js',
+			[ 'wp-polyfill-fetch' ],
+			self::get_asset_cache_buster(),
+			true
+		);
+
+		add_filter( 'script_loader_tag', [ $this, 'script_loader_tag' ], 10, 3 );
+
+		$dependencies = array();
+
+		if ( ! empty( $parsely_options['api_secret'] ) ) {
+			$dependencies[] = 'wp-parsely-api';
+			wp_localize_script(
+				'wp-parsely-api',
+				'wpParsely', // This globally-scoped object will hold our initialization variables
+				array(
+					'apikey' => $parsely_options['apikey'],
+				)
+			);
+		}
+
+		wp_enqueue_script(
+			'wp-parsely-tracker',
+			'https://cdn.parsely.com/keys/' . esc_url( $parsely_options['apikey'] ) . '/p.js',
+			$dependencies,
+			self::get_asset_cache_buster(),
+			true
+		);
+	}
+
+	public function script_loader_tag( $tag, $handle, $src ) {
+		$parsely_options = $this->get_options();
+		if ( in_array( $handle, [ 'wp-parsely', 'wp-parsely-tracker' ] ) ) {
+			// Have ClouldFlare Rocket Loader ignore these scripts:
+			// https://support.cloudflare.com/hc/en-us/articles/200169436-How-can-I-have-Rocket-Loader-ignore-specific-JavaScripts-
+			$tag = preg_replace( '/^<script src=/', '<script data-cfasync="false" src=', $tag );
+		}
+
+		if ( $handle === 'wp-parsely-tracker' ) {
+			$tag = preg_replace( '/ id=(\"|\')wp-parsely-tracker-js\1/', ' id="parsely-cfg"', $tag );
```

Comment:
Do we know why this change of ID is needed?

Reply:
I really don't know. This is another case of trying to not break anything without understanding why it is the way it is.

@hbbtstar -- any thoughts as to the necessity of having particular `id` attributes to the `script` tags?